chrome/tweaks/sidebery-collapsed.css

```diff
@@ -0,0 +1,57 @@
+@-moz-document regexp("^moz-extension://.*?/sidebar/sidebar.html") {
```

Comment:
The contents of the collapsed file could probably just be added to the end of the `sidebery.css` file instead of having two separate files.

Reply:
Merged into sidebery.css